screens/auth/LoginScreen.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { StyleSheet, Text, View, TextInput, Button } from 'react-native';
+import { useForm, Controller } from 'react-hook-form';
+import firebase from '../../firebase/clientApp';
+
+const auth = firebase.auth();
+
+export const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    backgroundColor: 'white',
+    width: '100%',
+    height: '100%',
+    paddingHorizontal: '12%',
+  },
+  content: {
+    height: '80%',
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  input: {
+    marginVertical: 10,
+    border: '1px solid black',
+    borderRadius: 5,
+    padding: 5,
+  },
+  red: {
+    color: 'red',
+  },
+  title: {
+    marginBottom: 30,
+    textAlign: 'center',
+  },
+  button: {
+    width: '80%',
+  },
+});
+
+type LoginData = {
+  email: string;
+  password: string;
+};
+
+const LoginScreen = ({ navigation }) => {
```

Comment:
Not sure how to get around this Typescript error, but will research in meantime.